services/scabbard/libscabbard/src/service/consensus.rs

```diff
@@ -413,6 +413,7 @@ mod tests {
             VecDeque::new(),
             Some(Box::new(service_sender.clone())),
             peer_services.clone(),
+            "0".to_string(),
```

Comment:
What does this `"0"` mean? If it is a service id, it would be nice to use a string that indicates as much, e.g., "svc0" (to conform to the length requirement)

Reply:
Updated this test to use "svc_" service IDs.